llama_index/types.py

```diff
@@ -3,7 +3,8 @@
 from abc import abstractmethod
 
 
-RESPONSE_TEXT_TYPE = Union[str, Generator]
+StreamTokens = Generator[str, None, None]
```

Comment:
This name is slightly confusing (I had to come here to confirm it was a generator)

Maybe `StreamTokenGen`  is more clear on what it is? Or `StreamGenTokens` 🤷🏻‍♂️

Reply:
good point. Will change the generator types to `*Gen`